index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alazar Esheetu</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  </head>
+  <body>
+    <div class="contanier">
+      <nav class="nav-contanier">
+        <div class="logo">Alazar Eshetu</div>
+        <div>
+          <img class="menu-icon" src="Images/Icon - Menu.png" alt="menu-icon" />
+        </div>
+      </nav>
+      <div class="headline">
+        <div><h1>Hey There. I’m Alazar</h1></div>
+        <div><p class="head">I’am a Software Developer</p></div>
+        <div>
+          <p class="description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="link">
+          <div class="all raddit">
+            <img src="Images/raddit.png" alt="raddit-icon" />
+          </div>
+          <div class="all Linkdn">
+            <img src="Images/Linkdn.png" alt="linkedn-icon" />
+          </div>
+          <div class="all peace"><img src="Images/peace.png" alt="Icon" /></div>
+          <div class="all tewtter">
+            <img src="Images/twet.png" alt="tewtter-icon" />
+          </div>
+          <div class="all mail"><img src="Images/mail.png" alt="mail-" /></div>
+          <div class="illustration">
+            <img src="Images/Illustration.png" alt="" />
+          </div>

```

Comment:
- [ ] I think it would be better to use the background image from the CSS side to align it with the position property using the background-image property :+1: